test/quantization/test_qat.py

```diff
@@ -1685,6 +1685,7 @@ def test_qat_range_learning(self):
             range_learning=True,
             scale_precision=torch.float32,
             zero_point_precision=torch.float32,
+            is_symmetric=True,
```

Comment:
can you test both True and False here by adding parameterization to this test?

Here's an example: https://github.com/pytorch/ao/blob/853f87d1bd56d8d6c980a518cf9fa08fbb20135c/test/quantization/test_qat.py#L1516